addons/web/static/src/js/owl_compatibility.js

```diff
@@ -494,6 +492,19 @@ odoo.define('web.OwlCompatibility', function () {
             }
             parentChildren.push(this);
         }
+        /**
+         * Stores mount target and position at first mount. That way, when updating
+         * while out of DOM, we know where and how to remount.
+         * @see update()
+         * @override
+         */
+        async mount(target, options) {
+            if (options && options.position === 'self') {
+                throw new Error(`Unsupported position: "self" is not allowed for wrapper component.`);
+            }
+            this._mountArgs = arguments;
```

Comment:
fair enough, this means that whatever future API mount will take, we have to mount the compatibility layer with the exact same arguments
It does make sense.